library/cmac.c

```diff
@@ -877,6 +877,17 @@ static int cmac_test_wth_cipher( int verbose,
                 continue;
             }
 
+            /* When CMAC is implemented by an alternative implementation, 3DES
```

Comment:
I'd prefer to group the two blocks are most of the code is common.

Reply:
Done